.github/workflows/stale_labeler.yml

```
@@ -0,0 +1,20 @@
+name: "Stale Labeler"
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)